src/www/vpn_ipsec_pubkeys.php

```diff
@@ -0,0 +1,203 @@
+<?php
+
+/*
+ * Copyright (C) 2019 Pascal Mathis <mail@pascalmathis.com>
```

Comment:
You probably copied a file, I’m not going to accept this as is.

Reply:
The required includes for \phpseclib\Crypt\RSA() are included earlier in the same file (see line 33 - 36) and the code was tested on a OPNsense machine running the latest release with `make mount`. phpseclib itself is already a dependency of the existing source code, specifically in `system_certmanager.php`. I made sure to only include files which are actually required and various public and private keys worked fine with the current list of includes.

Reply:
Good catch. While I did not copy one-by-one, I based various snippets / methods on the IPsec Mobile Key implementation and forgot to add them to the header. I will add Deciso B.V. and Manuel Kasper to the copyright notice once the other discussion points are finished.